### ISSUE TITLE
when i load a bninception model in tsn？  some wrong happen.

### ISSUE THREAD

**Issue by poppingcode** [yjxiong/tsn-pytorch#40]
i load a bninception model---https://github.com/Cadene/pretrained-models.pytorch/tree/master/pretrainedmodels/models
 It has been pre trained.

when i load a bninception model.
After dozens of epoch, loss has been stuck in a large number.
i don't know how to solve it ,please help me ,or  Or put forward a bit of thought。
thanks！！！！！

**Comment by poppingcode** [comment]
i load a bninception model---https://github.com/Cadene/pretrained-models.pytorch/tree/master/pretrainedmodels/models
It has been pre trained.

when i load a bninception model in tsn---https://github.com/yjxiong/tsn-pytorch/
After dozens of epoch, loss has been stuck in a large number.
i don't know how to solve it ,please help me ,or Or put forward a bit of thought。
thanks！！！！！

**Comment by yjxiong** [comment]
There are ImageNet pretrained models provided in this repo. You are using them if you follow the instructions in README. 
I suggest you start with them.

**Comment by sucaohan** [comment]
@poppingcode sorry，I have met same problem,have you solved it?
